src/components/Shows/NewShowForm/NewShowForm.js

```diff
@@ -0,0 +1,143 @@
+import React, { Component } from 'react'
+import PlacesAutocomplete, { geocodeByAddress, getLatLng } from 'react-places-autocomplete'
+import AddShowButton from './AddShowButton'
+
+const INITIAL_STATE = {date: '', venue: '', time: ''}
+
+class NewShowForm extends Component {
+  constructor (props) {
+    super(props)
+    this.state = INITIAL_STATE
+    this.onAutocompleteChange = (venue) => this.setState({...this.state, venue})
```

Comment:
why declare this method here? if it's moved outside of the constructor, and initial state is declared as a class property as well, the constructor can be removed.

Reply:
Good call. This was initially copied from google places api integration and I forgot to clean up